lib/config/config.go

```diff
@@ -9,29 +9,51 @@ import (
 )
 
 
+func CompileAncestorBranches(branchName string) (result []string) {
```

Comment:
There isn't really any calculation either. Compile has multiple definitions:

> 1. produce (something, especially a list, report, or book) by assembling information collected from other sources. "the local authority must compile a list of taxpayers"
synonyms: assemble, put together, make up, collate, compose, organize, arrange
> 2. COMPUTING (of a computer) convert (a program) into a machine-code or lower-level form in which the program can be executed.

 This is what it was in the bash src named it and I prefer it to calculate.